README.md

```diff
@@ -1,88 +1,89 @@
-# CAR Mirror over HTTP v0.1.0
+# CAR Mirror over HTTP v0.2.0
 
 ## Authors
 
 * [Brooklyn Zelenka], [Fission]
 
 ## Dependencies
 
+- [CAR Mirror]
 - [CARv1]
 - [CBOR]
 - [HTTP/2]
 - [IPLD Schema]
 
 ## Language
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119].
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14] when, and only when, they appear in all capitals, as shown here.
 
-# 0. Abstract
+# 0 Abstract
 
-CAR Mirror is transport agnostic, and MAY be implemented with a RESTful HTTP client/server. This specification describes the protocol for CAR Mirror over HTTP.
+This specification describes the protocol for synchronous [CAR Mirror] over [HTTP/2].
 
-# 1. Pull
+# 1 Introduction
 
-## 1.1 Endpoint
+[CAR Mirror] describes a generic method extending CAR files to efficiently mirror some IPLD graph state on a peer machine. It is transport agnostic.
+
+HTTP is a very mature protocol that connects the applciations and devices that developers — especially web developers — use most fequently. This spec defines how to perform multi-round, syncronous CAR Mirror over HTTP/2.
```

Comment:
Next step for md-spellcheck-action definitely is creating PR review comments with typo fix suggestions :thinking:

Reply:
That would be awesome, yeah

Reply:
@matheus23 the other thing that I _really_ want is a way to import external (via URL) custom dictionaries. The current copy/paste situation means that if I update one doc, I have to update _all_ of them, it's annoying when I have to add the same terminology several times, etc etc. I know, I know: "contributions welcome" 😉